.github/workflows/meta-labgrid.yml

```diff
@@ -45,7 +45,23 @@ jobs:
         run: |
           source poky/oe-init-build-env build
           bitbake python3-labgrid
-      - name: Build tools
+      - name: Build sispmctl
         run: |
           source poky/oe-init-build-env build
-          bitbake python3-pyserial sispmctl
+          bitbake sispmctl
+      - name: Build pyserial
+        run: |
+          source poky/oe-init-build-env build
+          bitbake python3-pyserial
```

Comment:
pyserial is a dependency of labgrid (built earlier). We should drop this.

Reply:
Makes sense, I've force-pushed a new version now that drops the pyserial build.